src/util.js

```diff
@@ -0,0 +1,21 @@
+import { Console } from "@woowacourse/mission-utils";
+
+export const formatErrorMessage = (error) => `[ERROR] ${error}`;
+
+export const runRace = (cars, inputRound) => {
+    for (let i = 0; i < inputRound; i++) {
+        cars.map((car) => car.move());
+        Console.print("\n");
+    }
+};
+
+export const getWinners = (cars) => {
+    const maxScore = cars.reduce((max, car) => Math.max(max, car.moveScore), 0);
+
+    let winners = [];
+    cars.map((car) => {
+        if (car.moveScore == maxScore) winners.push(car.name);
+    });
```

Comment:
map함수는 맵핑된 배열을 반환하므로, winner.push를 따로 하기보다,
```suggestion
const winners = cars.map((car)=> car.moveScore === maxScore));
```

이렇게 해도 될것 같아요.:)

Reply:
사용할 배열을 미리 선언하고 접근하다보니 이 부분을 놓쳤던 것 같아요. 
짚어주셔서 감사합니다!